pkg/trace/filters/replacer.go

```diff
@@ -60,9 +58,10 @@ func (f Replacer) ReplaceStatsGroup(b *pb.ClientGroupedStats) {
 			b.Resource = re.ReplaceAllString(b.Resource, str)
 			fallthrough
 		case "http.status_code":
-			strcode := re.ReplaceAllString(strconv.Itoa(int(b.HTTPStatusCode)), str)
-			if code, err := strconv.Atoi(strcode); err == nil {
-				b.HTTPStatusCode = uint32(code)
+			for _, t := range b.Tags {
+				if t.Name == "http.status_code" {
```

Comment:
We were using "HTTPStatusCode" in the tracer - is this what you need the tag to be named?